hooks/boot/13-tuning.rb

```diff
@@ -1,7 +1,7 @@
 if has_custom_fact?('tuning')
   # TODO: automatically get the tuning sizes - standard doesn't exist and only
   # loads base. The rest maps to config/foreman.hiera/tuning/sizes/$size.yaml
-  TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
+  TUNING_SIZES = ['development', 'default', 'medium', 'large', 'extra-large', 'extra-extra-large'].freeze
```

Comment:
My reasoning for the previous code style was that you always list `default` as the first entry, then all the non-default sizes. I think this is something we should keep since it's user facing (when they run with `--help`).

Reply:
Ah ha, should I put development at the end then to put it furthest away?

Reply:
Sounds good to me.